package.json

```diff
@@ -1,63 +1,42 @@
 {
-  "name": "js-craftcamp-website",
-  "version": "2.0.0",
-  "description": "Website for the JSCC (jscraftcamp.org)",
-  "engines": {
-    "node": ">= 18.0.0"
-  },
-  "dependencies": {
-    "classnames": "2.3.2",
-    "gatsby": "5.8.0",
-    "gatsby-plugin-image": "3.8.0",
-    "gatsby-plugin-manifest": "5.8.0",
-    "gatsby-plugin-offline": "6.8.0",
-    "gatsby-plugin-react-helmet": "6.8.0",
-    "gatsby-plugin-react-svg": "3.3.0",
-    "gatsby-plugin-sass": "6.8.0",
-    "gatsby-plugin-sharp": "5.8.0",
-    "gatsby-source-filesystem": "5.8.0",
-    "gatsby-transformer-json": "5.8.0",
-    "gatsby-transformer-sharp": "5.8.0",
-    "modern-normalize": "1.1.0",
-    "prop-types": "15.8.1",
-    "react": "18.2.0",
-    "react-d3-cloud": "1.0.6",
-    "react-dom": "18.2.0",
-    "react-helmet": "6.1.0",
-    "sass-mq": "6.0.0"
-  },
-  "devDependencies": {
-    "@storybook/addon-essentials": "6.5.16",
-    "@storybook/builder-webpack5": "6.5.16",
-    "@storybook/manager-webpack5": "6.5.16",
-    "@storybook/preset-scss": "1.0.3",
-    "@storybook/react": "6.5.16",
-    "gatsby-plugin-netlify": "5.1.0",
-    "hamjest": "3.7.3",
-    "hygen": "6.2.11",
-    "mocha": "10.2.0",
-    "prettier": "2.8.7",
-    "recursive-readdir-sync": "1.0.6",
-    "sass": "1.60.0",
-    "svg-sprite-loader": "6.0.11",
-    "svgo-loader": "4.0.0"
-  },
-  "scripts": {
-    "develop": "gatsby develop",
-    "format": "prettier --write src",
-    "start": "npm run develop",
-    "serve": "gatsby serve",
-    "prebuild": "mkdir -p ./public",
-    "build": "gatsby build",
-    "test": "mocha test",
-    "build-storybook": "STORYBOOK_BUILD=true build-storybook",
-    "storybook": "STORYBOOK_BUILD=true start-storybook -p 6006",
-    "prestorybook": "npm run extract-gatsby-meta-for-storybook",
-    "prebuild-storybook": "npm run prestorybook",
-    "extract-gatsby-meta-for-storybook": "mkdir -p public/page-data/sq/d ; node ./scripts/extract-gatsby-meta-for-storybook.js public/page-data/sq/d/3537562275.json",
-    "hygen": "hygen"
-  },
-  "author": "Michael Zoidl, Robert Hostlowsky and others",
-  "license": "MIT",
-  "repository": "https://github.com/jscraftcamp/website.git"
+	"name": "website",
+	"version": "0.0.1",
+	"private": true,
+	"engines": {
+		"node": ">= 18.0.0"
+	},
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
+		"test": "pnpm run test:unit",
+		"test:e2e": "playwright test",
```

Comment:
If there are no real ui e2e tests, I suggest to remove that and the additional playwright dependencies.
```suggestion
```